server/statistics/statistics.go

```diff
@@ -0,0 +1,164 @@
+package statistics
+
+import (
+	"sync"
+	"time"
+	"container/ring"
+	
+	"github.com/ubports/ubuntu-push/logger"
+)
+
+type StatsValue struct {
+	val5min int32
+	val60min *ring.Ring
+	val1day *ring.Ring
+	val7day *ring.Ring
+}
+
+func NewStatsValue() *StatsValue {
+	result := &StatsValue {
+	}
+	result.val60min = ring.New(12)
+	for i := 0; i < result.val60min.Len(); i++ {
```

Comment:
Are you sure you need to do these initializations? I would think that they are not needed: the rings can be kept empty, and filled as you get the data.

Reply:
Yes true, I was thinking that this creates a problem but it could work...